compiler/include/docsDriver.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2004-2015 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "arg.h"
+#include "misc.h"
+
+extern bool fDocs;
+extern bool fDocsPrintHelp;
+
+// FIXME: Comment out this option until it is re-enabled.
+//        (thomasvandoren, 2015-03-09)
+extern bool fDocsAlphabetize;
+
+extern char fDocsCommentLabel[256];
+extern char fDocsFolder[256];
+extern bool fDocsTextOnly;
+extern char fDocsSphinxDir[256];
+
+// FIXME: This is always true, so propagate that to the one spot it is used
+//        (thomasvandoren, 2015-03-08)
```

Comment:
This should really be a TODO on deciding whether we want a flag to ignore externs entirely.  Brad and I had a discussion after I had mostly implemented it (because I misinterpreted an action item), so we decided to leave it commented out in the code base and get back to the discussion after the release.